### PR TITLE
[PRD] Deploying CSS cleanup

### PR DESCRIPTION
**Pull request by pdelboca**
This PR is to deploy to PRD all our CSS cleanup.

# Notes
 - Removes all obsolete CSS codebase
 - Documents our Tailwind workflow
 - Fixes some classes that where not being discovered by Tailwind (#953 )
 - Fix search header icon: (#952 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,16 +106,17 @@ Same for staging.
...
-The css build is done by `vite` and the configuration files for it are `vite.config.js`, `tailwind.config.cjs` and `postcss.config.cjs`.
+The css build is done by `PostCSS` and the configuration files for it are `tailwind.config.cjs` and `postcss.config.cjs`.
 
 Running `npm run build` will compile our main `styles.css` file and place it in `static/css/styles.css`. (It then will be collected by 
 Django when building the Dockerfile)
 
-**Important: ** If you use a new tailwind class in some plugin, this class must be added somewhere on the file `templates/styles.html`, 
-otherwise the class will not work when used on the plugin.
+**Remember:** Tailwind CSS works by scanning all of our HTML files, JavaScript components, and any other templates
+ for class names to generate `styles.css`. If you ar adding new files, make sure to update the `content` field `tailwind.config.cjs`.
```
Comment (shashigharti) [inline README.md:118]: Typo near "If you ar"

File: README.md [hunk 2]
```diff
@@ -88,22 +98,69 @@ In staging and production, the uploaded files are hosted on Google Cloud Storage
...
-## Frontend and Static Assets
+**Note:** A dump of the production database will require all the media files of production.
+Same for staging.
+
+
+## Frontend
+
+### CSS
+
+Currently we are using Tailwind, PostCSS and Vite as our CSS stack. Please check the documentation to get a sense
```
Comment (shashigharti) [inline README.md]: I think we have removed Vite

**Review by shashigharti — COMMENTED** [review]
Looks good to me!. Added some comments on typos.

**Review by avdata99 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pdelboca) [reply README.md:118]: Thanks! I'll address it in another PR so I don't need to re-approve and merge to staging another PR. :)

**Pull request merged**